### PR TITLE
781 nft bridge

### PR DESCRIPTION
**Pull request by bilbolPrime**
### All Submissions

* [ ] Can you post a screenshot of your changes (if applicable)?

### New Feature Submissions

* [ ] Does this relate to an existing issue, or has it been talked about prior to being done (if a major change)?

### Notes

Please note, your code must pass all tests and lint checks before it can be merged.

### PR Description

final touches

### CODE REVIEW

**Comment by kevinunger** [comment]
@bilbolPrime 
Btn is now disabled, if transfered weapon is selected and status is 'done' or 'no transfer' or a transfer is processing/pending.
https://github.com/CryptoBlades/cryptoblades/blob/9efe7f3f640b327abb1a5403d9ed9d79f5b49a7f/frontend/src/views/Bridge.vue#L64-L66

File: frontend/src/views/Bridge.vue [hunk 1]
```diff
@@ -144,48 +162,60 @@
...
+                <option v-for="character in incomingChars"
+                :value="character['7']"
                 :key="character['3']">
                   Character ID: {{ character['3'] }} from Chain:
                   {{supportedChains[supportedChainIds.indexOf(character['2'])]}}
                 </option>
               </select>
               <div class="p-2 text-center">
-                <b-button :disabled="characterIdToWithdraw == null" variant="primary"
+                <b-button :disabled="characterIdToWithdraw == ''" variant="primary"
```
Comment (Chriserus) [inline frontend/src/views/Bridge.vue:209]: <b-button :disabled="!characterIdToWithdraw" variant="primary"

File: frontend/src/views/Bridge.vue [hunk 2]
```diff
@@ -347,13 +383,15 @@ export default Vue.extend({
...
-    console.log('currentNet: ', env);
     const conf = config as any;
     for(let i = 0; i < this.supportedChains.length; i++){
       this.supportedChainIds.push(conf.environments[env].chains[this.supportedChains[i]].VUE_APP_NETWORK_ID);
     }
   },
-
+  async mounted(){
+    if (!this.contracts.NFTStorage) return;
+    this.bridgeFee = await this.contracts.NFTStorage.methods.getBridgeFee().call({ from: this.defaultAccount });
```
Comment (Chriserus) [inline frontend/src/views/Bridge.vue:393]: .call(defaultCallOptions(state));

Comment (Chriserus) [reply frontend/src/views/Bridge.vue:393]: Also, please extract this to store.ts, we shouldn't have contract calls in components

**Comment by kevinunger** [comment]
@Chriserus Ty for the review. Will fix tomorrow morning asap

File: frontend/src/views/Bridge.vue (hunk 2 above)
Comment (Chriserus) [reply frontend/src/views/Bridge.vue:403]: As discussed, we'll leave it here, as we don't have a better solution on hand.

**Comment by Chriserus** [comment]
LGTM, thanks

**Review by Chriserus — APPROVED** [review]

**Review by kcper — APPROVED** [review]
Looks good, added `await` to calls in migration.

**Pull request merged**